src/sentry/static/sentry/app/components/dashboards/widgetQueryForm.tsx

```diff
@@ -80,12 +80,13 @@ class WidgetQueryForm extends React.Component<Props> {
           {canRemove && (
             <Button
               data-test-id="remove-query"
-              size="zero"
-              borderless
+              size="small"
+              priority="danger"
               onClick={this.props.onRemove}
               icon={<IconDelete />}
-              title={t('Remove this query')}
-            />
+            >
+              {t('Remove Overlay')}
```

Comment:
The big red button is a bit jarring. Perhaps we can come up with another visual treatment that helps make each overlay/query visually separate so the form doesn't feel so long.

Reply:
I'll ask @doralchan for guidance here.

Reply:
@doralchan mentioned that she'll follow up with a PR for clean up on Dashboards.